charts/portworx/templates/storage-cluster.yaml

```diff
@@ -74,7 +74,7 @@ spec:
   {{- end }}
 
   storage:
-    {{- if not (eq $drives "none") }}
+    {{- if or (hasPrefix "/" $drives) (contains ";/" $drives) }}
```

Comment:
We should avoid creating a `spec.storage` section if cloud storage specs are being used. I guess in the operator code, it would ignore the cloudstorage spec if storage section is present.

Reply:
actually i looked at code, if storage section is not nil, operator will never use cloudStorage section. let me fix this